uli/README.md

```diff
@@ -7,11 +7,13 @@
 ## Usage
 
 ```
-fin uli [-s] [@drushsitealias] [ drush uli options ]
+fin uli [-s] [-i] [@drushsitealias] [ drush uli options ]
 ```
 
 * `-s` outputs the login url with https instead of default of https
```

Comment:
This wasn't part of the update, but should be updated. This should read:
 `-s` outputs the login url with https instead of default of http

Reply:
I pushed a commit to this feature branch that fixes this typo.